src/utils.ts

```diff
@@ -380,6 +380,16 @@ export const setMaxStalePeriod = async (
         normalTimelock,
         maxStalePeriodInSeconds,
       );
+
+      if (underlyingAsset.address === "0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB") {
+        await setMaxStalePeriodInChainlinkOracle(
+          tokenConfig.oracles[1],
```

Comment:
Why do we need to configure maxStalePeriod also for the pivot oracle?

Reply:
Just to confirm, `checkIsolatedPoolsComptroller` fails even without this VIP, right? Do you know where we have underlying asset == BNB (and not WBNB) in IL?